### PR TITLE
Create unit test for pkg/time/time.go

### PR DESCRIPTION
**Pull request by egurnov**
**What this PR does / why we need it**: Resolves openebs/litmus#300 by adding tests for pkg/time/time.go

**Which issue this PR fixes**: openebs/litmus#300

**Special notes for your reviewer**:
I had to provide a way to replace calls to actual time.Sleep to test longer duration values.

### CODE REVIEW

**Comment by egurnov** [comment]
Looks like build environment is broken.
```
make: gometalinter: Command not found
```
My code has nothing to do with gometalinter.

**Comment by ksatchit** [comment]
The travis build has been fixed. Once we done with/agree to the changes, should be ok to merge!

File: pkg/time/time.go [hunk 1]
```diff
@@ -36,6 +36,8 @@ func WaitFor(duration string) (err error) {
 		return
 	}
 
-	time.Sleep(interval)
+	sleep(interval)
 	return
 }
+
+var sleep = time.Sleep
```
[the remark this answers is not in the document]
Comment (egurnov) [reply pkg/time/time.go]: To the best of my knowledge, there are no clear instructions for this.

There are a bunch of packages with provide wrappers for time package with rich mocking capabilities. 

https://github.com/cloudfoundry/clock
https://github.com/jonboulle/clockwork
https://github.com/benbjohnson/clock
https://github.com/101loops/clock
https://github.com/WatchBeam/clock
https://github.com/smartystreets/clock

However using one of them for the task at hand would be an overkill, I suppose.
And then again: 
> A little copying is better than a little dependency.
-- Rob Pike

And I'm not even copying anything. 

Here's a reddit thread on the topic: https://www.reddit.com/r/golang/comments/530cqp/how_to_mock_time_for_testing_purposes/ 
One of the comments mentions a talk from Google I/O 2014 which illustrates a similar technique: https://youtu.be/ndmB0bj7eyw?t=32m

As for unintentional assignment, both `sleep` and `swapSleep` are not exported, so you can only mess with them from inside the package. Also the reason I put tests in the same package, not `time_test`.

[the remark this answers is not in the document]
Comment (AmitKumarDas) [reply pkg/time/time.go]: Thank you @egurnov . Looks fine to me.

**Pull request merged**